src/Makefile.qt.include

```diff
@@ -236,27 +236,19 @@ RES_ICONS = \
   qt/res/icons/connect2_16.png \
   qt/res/icons/connect3_16.png \
   qt/res/icons/connect4_16.png \
-  qt/res/icons/cpumined_blue.png \
   qt/res/icons/debugwindow.png \
   qt/res/icons/edit.png \
   qt/res/icons/editcopy.png \
   qt/res/icons/editpaste.png \
   qt/res/icons/ex.png \
   qt/res/icons/export.png \
   qt/res/icons/filesave.png \
-  qt/res/icons/history.png \
   qt/res/icons/gold_cpumined.gif \
-  qt/res/icons/gold_cpumined.png \
   qt/res/icons/key.png \
-  qt/res/icons/lock_closed.png \
```

Comment:
Aren't these still used?

> qt/coincontroldialog.cpp:    contextMenuItem->setIcon(COLUMN_CHECKBOX, QIcon(":/icons/lock_closed"));
> qt/coincontroldialog.cpp:                itemOutput->setIcon(COLUMN_CHECKBOX, QIcon(":/icons/lock_closed"));

Reply:
That portion of code is commented out in coincontroldialog.cpp . Why is it commented out @skcin  ?

Reply:
Actually, checking the blame it looks like it was never used in the first place.

Reply:
Aye, it's used in more places like `bitcoingui.cpp` though.

Reply:
I can only see the .svg files in the bitcoingui.cpp , where /icons/lock_closed_"+sSheet imports the correctly styled .svg . Do you have a line number?